docs-ref-services/data-schemaregistry-readme-pre.md

```diff
@@ -132,8 +132,10 @@ String schemaContent = "{\n"
     + "        }\n"
     + "    ]\n"
     + "}";
+Schema.Parser schemaParser = new Schema.Parser();
+Schema schema = schemaParser.parse(schemaContent);
 SchemaProperties schemaProperties = schemaRegistryClient.registerSchema("{schema-group}", "{schema-name}",
-    schemaContent, SerializationType.AVRO);
+    schema.toString, SerializationType.AVRO);
```

Comment:
This change should not be done in this file. This is generated from this README file - https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/schemaregistry/azure-data-schemaregistry/README.md

So, all updates have to be done to the source README and the docs will get updated.

Reply:
Thank you for the review @srnagar, I'm closing the pull request.